modules/fuzztest/20240813.0/patches/update_module_bazel_version.patch

```diff
@@ -0,0 +1,26 @@
+--- MODULE.bazel
++++ MODULE.bazel
+@@ -1,6 +1,6 @@
+ module(
+     name = "fuzztest",
+-    version = "head",
++    version = "20240813.0",
+     repo_name = "com_google_fuzztest",
+ )
+ 
+@@ -46,7 +46,6 @@ bazel_dep(
+ bazel_dep(
+     name = "rules_proto",
+     version = "5.3.0-21.7",
+-    dev_dependency = True,
+ )
+ 
+ bazel_dep(
+@@ -51,7 +51,6 @@
+ bazel_dep(
+     name = "riegeli",
+     version = "0.0.0-20240606-973b6f0",
+-    dev_dependency = True,
+     repo_name = "com_google_riegeli",
+ )
+ 
```

Comment:
This needs to contain a space. You might have to teach your editor to keep lines only consisting of a space in `.patch` files.